Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Commands/Settings/RunCommandSettings.cs

```diff
@@ -36,22 +36,6 @@ public sealed class RunCommandSettings : CommandSettings
     [Description("Disable auto launching the test tool's web interface in a browser.")]
     public bool NoLaunchWindow { get; set; }
 
-    /// <summary>

```

Comment:
These were leftover from the old version of the tooling and are not used in the new version.